text/survey_analysis_report2.md

```diff
@@ -0,0 +1,191 @@
+Here we present the results of an initial survey run in three neuroscience research consortium (LINK).
+
+
+We collected the feedback from 51 neuroscientists presented with 2 initial drafts,
+analysed their responses,
+and build a second template draft based on this feedback that is presented here.
+This new template takes three levels of data organisation into account (experiment, project, laboratory),
+while staying relatively simple and flexible, especially when implemented it in a git workflow (using the GIN-tonic tool we are developing).
+Researchers from all research domains are now invited to review this draft in a second survey (run until mid January 2021, LINK).  
+
+
+
+# Introduction
+
+Researchers spend time every day doing file management on their computers (creating, downloading, naming,  moving,  saving,  copying,  reviewing,  navigating, searching for, sharing, and deleting files and folders).
+While many different initiatives and tools tried to improve file management (using tags, databases and search algorithms), the use of folder tree structure appeared to be unavoidable [@dinneenUbiquitousDigitalFile2020]. 
+While both reproducible research and data management advocates and experts are advising researchers to design a folder organisation structure ([@LibraryCarpentryLcfairresearch2019], [@theturingwaycommunityTuringWayHandbook2019]),
+only few actually provide examples or attempt to homogenize the structure (but see [@vuorreSharingOrganizingResearch2020] and [@wilsonGoodEnoughPractices2017] for exceptions).
+
+We are data managers of different institutions working with neurobiologists, and we teamed up with the NFDI-neuro community to develop a new strategy to help researchers with their data management. 
+We hypothesize that implementing a folder tree structure using a template could help researchers collaborate on their projects,
+and manage their data and files better.
+In a first step, 
+we designed two templates (fig. 1) and a survey to guide our efforts.
+We are presenting here the analysis of the survey answer and a novel version of the template in accordance with the received feedback.
+This blog ends by outlining the gin-tonic application, that brings some technical solution (based on the git submodule technology) to add flexibility and ease of use in the template. 
+
+> *How do you organise your digital files for your research projects ?* 
+>
+> I organise my file using the GIN research repository (version 1).
+
+# The survey
+
+## Two template structure
+
+In order to obtain a practical flair in comments and feedback received,
+we provided two templates, with a similar number of folders,
+but organized differently.
+The 5_top structure was more hierarchical,
+while the 9_top structure was flatter (Fig. 1).
+In addition, we asked researchers to browse the folder tree while asking them to place or find specific files,
```

Comment:
Then, we asked researchers to browse the folder tree while asking them to place or find specific files,